lib/routes/sustainabilitymag/articles.ts

```diff
@@ -0,0 +1,98 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import logger from '@/utils/logger';
+import { parseDate } from '@/utils/parse-date';
+import puppeteer from '@/utils/puppeteer';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: '/articles',
+    name: 'Articles',
+    url: 'sustainabilitymag.com/articles',
+    maintainers: ['@mintyfrankie'],
+    categories: ['other'],
+    example: '/sustainabilitymag/articles',
+    radar: [
+        {
+            source: ['https://sustainabilitymag.com/articles'],
+            target: '/sustainabilitymag/articles',
+        },
+    ],
+    features: {
+        requireConfig: false,
+        requirePuppeteer: true,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    handler,
+};
+
+async function handler() {
+    const baseURL = `https://sustainabilitymag.com`;
+    const feedURL = `${baseURL}/articles`;
+    const feedLang = 'en';
+    const feedDescription = 'Sustainability Magazine Articles';
+
+    const browser = await puppeteer();
+    const page = await browser.newPage();
+    await page.setRequestInterception(true);
+    page.on('request', (request) => {
+        request.resourceType() === 'document' ? request.continue() : request.abort();
+    });
+    logger.http(`Requesting ${feedURL}`);
+    await page.goto(feedURL, {
+        waitUntil: 'domcontentloaded',
+    });
+    const response = await page.content();
+    const $ = load(response);
+
+    const list = $('#content > div > div > div:nth-child(4) > div > div > div > div > div > div:nth-child(2) > div.infinite-scroll-component__outerdiv > div > div > div')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            const title = item.find('h3').first().text();
+            const a = item.find('a').first();
+            return {
+                title,
+                link: `${baseURL}${a.attr('href')}`,
+            };
+        });
+
+    page.close();
```

Comment:
Unnecessary use of puppeteer.
You can obtain the list of latest articles from their GraphQL endpoint `https://sustainabilitymag.com/graphql` with no difficulties.

Reply:
> Unnecessary use of puppeteer. You can obtain the list of latest articles from their GraphQL endpoint `https://sustainabilitymag.com/graphql` with no difficulties.

Hey @TonyRL oups nice catch! Didn't manage to find out.

Will rework the router then, thanks!